alert-database/src/test/java/com/synopsys/integration/alert/database/api/mock/MockSettingsKeyRepository.java

```diff
@@ -0,0 +1,152 @@
+package com.synopsys.integration.alert.database.api.mock;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.springframework.data.domain.Example;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import com.synopsys.integration.alert.database.settings.SettingsKeyEntity;
+import com.synopsys.integration.alert.database.settings.SettingsKeyRepository;
+
+public class MockSettingsKeyRepository implements SettingsKeyRepository {
```

Comment:
We created these "mock" repositories for test purposes so that we can control the functionality within repository and do not have to mock out the returns from each of these methods since these repositories were easier to implement than others.